ecleankernel/file.py

```diff
@@ -126,11 +135,13 @@ def decompress_raw(self) -> bytes:
                     reader = zstandard.ZstdDecompressor().stream_reader(f)
                     decomp = b''
                     while True:
-                        chunk = reader.read(1024*1024)
+                        chunk = reader.read(1024 * 1024)
                         if not chunk:
                             break
                         decomp += chunk
                     return decomp
+                elif comp == 'lzma':
+                    return self.decompress_lzma(f.read())
```

Comment:
I think you've reached the point when you can do:

```suggestion
                    return LZMADecompressor().decompress(f.read())
```

Reply:
Plus maybe a short comment above that telling why it's needed.

Reply:
Yess, thank you.